doc/faq.rst

```diff
@@ -282,11 +282,17 @@ documentation <https://docs.python.org/3/library/multiprocessing.html#contexts-a
 
 Why does my job use more cores than specified with n_jobs under OSX or Linux?
 -----------------------------------------------------------------------------
-This can happen when the parallelized computation is handled outside of Python by another library,
-such as MKL or OpenBLAS. In that case, the value of ``n_jobs`` is ignored by the library,
-and the number of threads must be specified via an environment variable.
-For example, to set the maximum number of threads to some integer value ``N``,
-the following environment variables must be set:
+
+This happens when vectorized numpy operations are handled outside of Python
```

Comment:
I'd rather remove "outside of python". In this sense much of python happens outside python

Reply:
Fair enough! ;)

Reply:
Thank you @jnothman for guiding me through my first PR!